addons/hw_drivers/iot_handlers/interfaces/DisplayInterface_L.py

```diff
@@ -22,28 +26,41 @@ def get_devices(self):
 
         if screeninfo is None:
             # On IoT image < 24.08 we don't have screeninfo installed, so we can't get the connected displays
-            # We return a single display with x_screen = 0, to open a browser anyway, in case one is connected
-            display_identifier = 'hdmi_0'
-            display_devices[display_identifier] = {
-                'identifier': display_identifier,
-                'name': 'Display - ' + display_identifier,
-                'x_screen': '0',
-            }
-            return display_devices
+            # We use old method to get the connected display
+            hdmi_ports = {'hdmi_0': 2, 'hdmi_1': 7} if 'Pi 4' in GPIO.RPI_INFO.get('TYPE') else {'hdmi_0': 2}
+            try:
+                for x_screen, port in enumerate(hdmi_ports):
+                    if Vcgencmd().display_power_state(hdmi_ports.get(port)) == 'on':
+                        display_devices[port] = self._add_device(port, x_screen)
+                return display_devices
```

Comment:
Shouldn't we return a dummy display in case no displays are found here?
```suggestion
            return display_devices if display_devices else dummy_display
```